### PR TITLE
docs(*): add chart submission guidelines to contrib guide, update readme

### LINKED ISSUE #20: add chart submission guidelines
issue is based off of @pwittrock's suggestion. 
Feel free to add more here @pwittrock

### PR DESCRIPTION
**Pull request by prydonius**
/cc @migmartri @michelleN @viglesiasce

closes #20

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,11 +15,53 @@ Follow either of the two links above to access the appropriate CLA and instructi
...
+    * Customization: explaining options in `values.yaml` and their defaults
+* Must include a short `notes.txt`, including:
+    * Any relevant post-installation information for the Chart
+    * Instructions on how to access the application or service provided by the Chart
+
+#### Merge approval and release process
+
+A Kubernetes Charts maintainer will review the Chart submission, and start a validation job in the CI to verify the technical requirements of the Chart. A maintainer may add "LGTM" (Looks Good To Me) or an equivalent comment to indicate that a PR is acceptable. Any change requires at least one LGTM. No pull requests can be merged until at least one maintainer signs off with an LGTM.
+
+Once the Chart has been merged, the release job will automatically run in the CI to package and release the Chart in the [`gs://kubernetes-chart` Google Storage bucket](https://console.cloud.google.com/storage/browser/kubernetes-charts/).
```
Comment (michelleN) [inline CONTRIBUTING.md]: gs://kubernetes-chart**s**

File: README.md [hunk 2]
```diff
@@ -2,24 +2,26 @@
...
 
-Take a look at the [alpine example chart](https://github.com/kubernetes/helm/tree/master/docs/examples/alpine) and the [nginx example chart](https://github.com/kubernetes/helm/tree/master/docs/examples/nginx) for reference when you're writing your first few charts..
+Take a look at the [alpine example chart](https://github.com/kubernetes/helm/tree/master/docs/examples/alpine) and the [nginx example chart](https://github.com/kubernetes/helm/tree/master/docs/examples/nginx) for reference when you're writing your first few charts.
 
 Before contributing a Chart, become familiar with the format. Note that the project is still under active development and the format may still evolve a bit.
 
-## Contributing a chart
+## Repository Structure
+
+This GitHub repository contains the source for the packaged and versioned charts released in the [`gs://kubernetes-chart` Google Storage bucket](https://console.cloud.google.com/storage/browser/kubernetes-charts/) (the Chart Repository).
```
Comment (michelleN) [inline README.md]: 👍

**Comment by michelleN** [comment]
looks great @prydonius - I have a few very minor comments.

**Comment by prydonius** [comment]
Thanks @michelleN, fixed the GS bucket name :)

**Pull request merged**